### DISCUSSION TITLE
authenticated-pages.md: I have one question with using the Option 2

### DISCUSSION THREAD

**Discussion by WindInMay524** [GoogleChrome/lighthouse#14299] [Q&A]
I want to check other urls  performance but when I try to report a performence report  ,the url is the same one all the time 
I'm not sure which url web performance I have reseived
![image](https://user-images.githubusercontent.com/38344643/185084290-19b08ed7-ca56-40c9-b2ec-f3f9631d6151.png)

**Comment by adamraine** [comment]
This is WAI but it's a problem we are attempting to smooth over #13706.

The URL at the top of the Lighthouse refers to the network request associated with the main HTML document.

However, JS on the page can change the URL in the address bar before the page finishes loading, this is what I suspect is happening on your page.

In the future, we plan to change the URL at the top of the report to account for changes made during the page load.
